### DISCUSSION TITLE
Toolbar at the bottom

### DISCUSSION THREAD

**Discussion by matthias-itb** [Alex-D/Trumbowyg#1345] [Q&A]
Hi,
is it possible to display the toolbar below the text area? 
(Request from a customer)

**Comment by Alex-D** [comment]
That's not supported in Trumbowyg, but feel free to customize the CSS to get that working for you :)

**Comment by CreativeSparkStudios** [comment]
You could always add this CSS to limit the height of the textarea so that the top toolbar stays visible as you scroll through the content: .trumbowyg-editor, .trumbowyg-box {
  max-height: 600px !important;
}
